backend/app/services/adapters/task_kinds/task_skills_resolver.py

```
@@ -131,14 +132,26 @@ def resolve_task_skills(db: Session, *, task_id: int, user_id: int) -> Dict[str,
     user_selected_skills = parse_additional_skill_names_from_labels(labels)
 
-    team = kindReader.get_by_name_and_namespace(
-        db, task_owner_id, KindType.TEAM, team_namespace, team_name
+    team = (
+        db.query(Kind)
+        .filter(
+            Kind.user_id == task_owner_id,
+            Kind.kind == KindType.TEAM.value,
+            Kind.namespace == team_namespace,
+            Kind.name == team_name,
+            Kind.is_active == True,
```

Ruff (0.15.18)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)